tests/phpunit/elementor/core/utils/import-export/test-site-settings.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
@davseve I don't see a need for this test because I already covered this in my test. The only added value would be if you can actually Act the whole import process...

Reply:
Test removed